selfdrive/ui/translations/auto_translate.py

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env python3
+
+import argparse
+import logging
+import os
+import sys
+import xml.etree.ElementTree as ET
+from typing import cast
+
+import requests
+
+MODEL = "gpt-4"
+
+OPENAI_API_KEY = os.environ["OPENAI_API_KEY"]
```

Comment:
if this isn't set, let's exit 1 and print a link to https://beta.openai.com/account/api-keys